src/test/versionbits_tests.cpp

```diff
@@ -305,16 +270,25 @@ static void check_computeblockversion(const Consensus::Params& params, Consensus
     BOOST_CHECK_EQUAL(ComputeBlockVersion(nullptr, params), VERSIONBITS_TOP_BITS);
 
     // always/never active deployments shouldn't need to be tested further
-    if (nStartTime == Consensus::BIP9Deployment::ALWAYS_ACTIVE) return;
-    if (nStartTime == Consensus::BIP9Deployment::NEVER_ACTIVE) return;
+    if (nStartTime == Consensus::BIP9Deployment::ALWAYS_ACTIVE ||
+        nStartTime == Consensus::BIP9Deployment::NEVER_ACTIVE)
+    {
+        BOOST_CHECK_EQUAL(min_activation_height, 0);
+        return;
+    }
```

Comment:
```diff
@@ -271,8 +367,7 @@ static void check_computeblockversion(const Consensus::Params& params, Consensus
 
     // always/never active deployments shouldn't need to be tested further
     if (nStartTime == Consensus::BIP9Deployment::ALWAYS_ACTIVE ||
-        nStartTime == Consensus::BIP9Deployment::NEVER_ACTIVE)
-    {
+        nStartTime == Consensus::BIP9Deployment::NEVER_ACTIVE) {
         BOOST_CHECK_EQUAL(min_activation_height, 0);
         return;
     }
```

Reply:
Personally I find it incredibly ugly to have the if-condition and the first if-branch continue with the same indentation as if they are one block. It might be good to update the clang-format to not do this.

Also this file is far from clang-format clean, so I am going to leave as is for now.

Reply:
@MarcoFalke  **CONCEPT ACK**